frontend/components/Cooperate.vue

```diff
@@ -4,6 +4,7 @@
       <img
         class="w-full h-full object-cover"
         src="../assets/images/cooperate.jpg"
+        alt="People discussing about the project"
```

Comment:
```suggestion
        alt=""
```
I don't think those alternative texts are valid for decorative content. Those descriptions have no functionality beyond allowing user to imagine how our site looks like. Legally blind person might not be thrilled to do that.

You need to consider that screen reader might be going through each image, preventing user from reaching important content of the site.

[This site](https://supercooldesign.co.uk/blog/how-to-write-good-alt-text) has good information about this issue in a nutshell.
For more in-depth information, you can head [here.](https://webaim.org/techniques/alttext/)

Reply:
All images must have alt attributes for better SEO.

Reply:
I've read a little more on the subject and it seems right if we don't want search engines to consider site as a spam. 
Though there's additional notice that it's best practice to embed decorative images into CSS, without using `img` html element. This way it doesn't clash with both SEO and accessibility, because the former requires human-like description, while the latter recommends avoiding decorative elements.
It even encourages adding more alt texts to forms, so the impaired person can select them with their navigation tools.

Reply:
Alternatively we can go with this approach for now, but we can later test how the website behaves with screen reader. It should be interesting to see how it behaves, maybe there won't be any troubles, but it's important. 

We have to remember that some universities might offer full support for disabled students, and providing native solution for accessibility software might be useful. It doesn't even have to be for legally blind people. Somebody might simply use text-to-speech to navigate the site for other reasons. There's whole set of things to consider when designing accessible site, like proper headings and usage of other html elements.

If there are problems with accessibility, it can be addressed in separate issue.